modules/nf-core/pretextsnapshot/main.nf

```diff
@@ -0,0 +1,31 @@
+process PRETEXTSNAPSHOT {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda (params.enable_conda ? "bioconda::pretextsnapshot=0.0.4" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/pretextsnapshot:0.0.4--h7d875b9_0':
+        'quay.io/biocontainers/pretextsnapshot:0.0.4--h7d875b9_0' }"
+
+    input:
+    tuple val(meta), path(pretext)
+
+    output:
+    tuple val(meta), path("*.png"), emit: picture
+    path "versions.yml"           , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+
+    """
+    PretextSnapshot $args -m $pretext
```

Comment:
Also missing `-o` and `--prefix`, the latter you need  to take the `prefix` script definition that normally omes from the module template [^1].

[^1]: https://nf-co.re/docs/contributing/modules#naming-conventions  point 5